samza-api/src/main/java/org/apache/samza/operators/windows/Windows.java

```diff
@@ -163,23 +169,25 @@ private Windows() { }
    *
    * <pre> {@code
    *    MessageStream<String> stream = ...;
-   *    BiFunction<String, Integer, Integer> maxAggregator = (m, c)-> Math.max(parseInt(m), c);
+   *    Supplier<Integer> initialValue = () -> 0;
+   *    FoldLeftFunction<String, Integer, Integer> maxAggregator = (m, c) -> Math.max(parseInt(m), c);
    *    MessageStream<WindowPane<Void, Integer>> windowedStream = stream.window(
    *        Windows.tumblingWindow(Duration.ofSeconds(10), maxAggregator));
    * }
    * </pre>
    *
    * @param duration the duration in processing time
```

Comment:
Nit: Why is this parameter named differently for keyed and unkeyed tumbling windows?